.github/workflows/wheels.yml

```diff
@@ -20,20 +20,20 @@ jobs:
         env:
           # Specify which Python versions to build wheels
           # https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
-          CIBW_BUILD: "cp37-* cp38-* cp39-* cp310-*"
-          # Skip 32 bit architectures, musllinux, and i686, and macOS x86_64 wheels for CP3.8 -- CP3.10
-          CIBW_SKIP: "*-win32 *-musllinux_x86_64 *_i686 cp38-macosx_x86_64 cp39-macosx_x86_64 cp310-macosx_x86_64"
+          CIBW_BUILD: "cp37-* cp38-* cp39-* cp310-* cp311-*"
```

Comment:
@raphaelreinauer feel free to remove python 3.7 as well. It is anyway not supported anymore.

Reply:
Also, this will make the CI pass and we can merge your PR

Reply:
I'll do. Should I add Python 3.12 as well?